src/seedu/addressbook/data/AddressBook.java

```diff
@@ -48,6 +48,13 @@ public AddressBook(UniquePersonList persons, UniqueTagList tags) {
         }
     }
 
+    /**
+     * Sort persons
```

Comment:
So we must be precise with the comments provided for the methods! Is this sorting of persons name, address or both?